addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/service/MainTVServerService.java

```diff
@@ -130,7 +132,7 @@ public void run() {
 
                     updateResourceState("MainTVAgent2", "GetCurrentBrowserURL", null);
                 } catch (Exception e) {
-                    logger.debug("Exception during poll : {}", e);
+                    reportError("Error occurred during poll", e);
```

Comment:
My TV behaviour doesn't always follow the "spec", e.g. some of the resources doesn't always return correct amount of parameters.

2017-06-18 11:20:50.921 [DEBUG] [e.s.i.t.upnp.UpnpIOServiceImpl:306  ] - Error reading SOAP response message. Can't transform message payload: org.jupnp.model.action.ActionException: The argument value is invalid. Invalid number of input or output arguments in XML message, expected 3 but found 1.
2017-06-18 11:20:50.992 [DEBUG] [e.s.i.t.upnp.UpnpIOServiceImpl:306  ] - Error reading SOAP response message. Can't transform message payload: org.jupnp.model.action.ActionException: The argument value is invalid. Invalid number of input or output arguments in XML message, expected 2 but found 1.

In this case TV only send correct amount of parameters when web browser is enabled.

Currently UpnpIOServiceImpl doesn't throw exceptions e.g. when there is wrong amount of parameters, but if UpnpIOServiceImpl behaviour is changed, then this change cause that TV is set to offline state.

Reply:
@martinvw, sorry I have missed you question/comment. I'm fine with that change.